Sources/Logging/Loggable.swift

```diff
@@ -10,15 +17,30 @@ protocol Loggable {
 }
 
 extension Loggable {
+    var date: Date? { nil }
+    var prefix: String? { nil }
     var cssClass: String? { nil }
 
     var logData: Data? {
         if let cssClass = cssClass {
-            return "<p class=\"\(cssClass)\">\(message)</p>".data(using: .utf8)
+            return "<p class=\"\(cssClass)\">\(logMessage)</p>".data(using: .utf8)
         } else {
             return message.data(using: .utf8)
         }
     }
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Logging/Loggable.swift;
  Line: 31;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)